modules/taskFunction.js

```diff
@@ -0,0 +1,171 @@
+import TODoTasks from './tasks.js';
+
+const task = document.getElementsByClassName('tasks')[0];
```

Comment:
```suggestion
const task = document.querySelector('.tasks');
```
**OPTIONAL**
Maybe its better to use querySelector and it will directly return the first item of the array